configure.ac

```diff
@@ -78,7 +78,8 @@ m4_define([libva_lt_age],
 m4_define([libdrm_version], [2.4])
 
 # Wayland minimum version number
-m4_define([wayland_api_version], [1.0.0])
+# 1.11.0 for wl_proxy_create_wrapper
+m4_define([wayland_api_version], [1.11.0])
```

Comment:
It's not just me compiling libva... it's our CI and entire automated infrastructure that will need amended.  It's not impossible to do, but needs to be considered when making these kinds of changes.  Anyhow, I'll work with maintainers to come to a decision about how we want to deal with these kinds of breakages/requirements.